src/kbmod/search/stack_search.cpp

```diff
@@ -256,17 +256,33 @@ std::vector<float> StackSearch::extract_psi_or_phi_curve(Trajectory& trj, bool e
     return result;
 }
 
+std::vector<std::vector<float> > StackSearch::get_psi_curves(std::vector<Trajectory>& trajectories) {
```

Comment:
Any reason why the input isn't `const &`?

Reply:
No particularly good reason. I made that change (and also updated some of the other functions).